### PR TITLE
[PB] Crash screen & OTA feedback

### PR DESCRIPTION
**Pull request by NiLuJe**
Much like on other platforms w/ an FBInk port ;).

Depends on https://github.com/koreader/koreader-base/pull/1192

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6686)

### CODE REVIEW

**Comment by NiLuJe** [comment]
Untested ;p.

If you require FBInk binaries to help test this, they're [here](https://github.com/NiLuJe/FBInk/pull/47#issuecomment-695152346) ;).

**Review by Frenzie — APPROVED** [review]
I assume the shell script is the same as on Kobo and it's really all about FBInk? ^_^

**Comment by NiLuJe** [comment]
Essentially, yes ;).

There are a few platform-specific stuff done differently, but hopefully I didn't botch that too much ;p.

File: platform/pocketbook/koreader.app [hunk 1]
```diff
@@ -76,11 +78,108 @@ if [ -e crash.log ]; then
...
+        ko_update_check
+    fi
+
     ./reader.lua "${args}" >>crash.log 2>&1
-    RETURN_VALUE=$(cat ${KO_EXIT_CODE})
+    RETURN_VALUE="$(cat ${KO_EXIT_CODE})"
+    rm -f "${KO_EXIT_CODE}"
+
+    # Did we crash?
+    if [ ${RETURN_VALUE} -ne 0 ] && [ ${RETURN_VALUE} -ne ${KO_RC_RESTART} ]; then
```
Comment (ezdiy) [inline platform/pocketbook/koreader.app]: On  crash, the file in ${KO_EXIT_CODE} will be always empty, so perhaps one could just do `if [ -e ${KO_EXIT_CODE} ]`. While these conditions will hold true in the empty case, it's more of a side-effect fluke than stating what really goes on explicitly.

Comment (NiLuJe) [reply platform/pocketbook/koreader.app]: Yeah, good point ;).

**Comment by NiLuJe** [comment]
Let's break all the things! (hopefully not ^^).

**Pull request merged**